Makefile

```diff
@@ -160,13 +166,12 @@ rm -rf $$TMP_DIR ;\
 endef
 
 DEPLOYMENT_DIR = $(PROJECT_DIR)/config/deploy
-DEPLOYMENT_FILE = $(DEPLOYMENT_DIR)/$(shell basename $(AUTHORINO_MANIFESTS))
+DEPLOYMENT_FILE = $(DEPLOYMENT_DIR)/manifests.yaml
 .PHONY: deploy-manifest
 deploy-manifest:
 	mkdir -p $(DEPLOYMENT_DIR)
-	curl -sSf $(AUTHORINO_MANIFESTS) > $(DEPLOYMENT_FILE) && sed -i '$${/^$$/d;}' $(DEPLOYMENT_FILE) && echo '---' >> $(DEPLOYMENT_FILE)
 	cd $(PROJECT_DIR)/config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMAGE) ;\
-	cd $(PROJECT_DIR) && $(KUSTOMIZE) build config/default >> $(DEPLOYMENT_FILE)
+	cd $(PROJECT_DIR) && $(KUSTOMIZE) build config/deploy > $(DEPLOYMENT_FILE)
```

Comment:
A lot safer this way.

TBH, I didn't know kustomize allowed to point to remote resources. Apparently since [March 2020](https://github.com/kubernetes-sigs/kustomize/pull/2167).